tests/integration/full-flow/test_harvest_full_flow.py

```diff
@@ -0,0 +1,72 @@
+from unittest.mock import patch
+
+from harvester.harvest import HarvestSource
+
+
+class TestHarvestFullFlow:
+    @patch("harvester.harvest.ckan")
+    def test_harvest_single_record_created(
+        self,
+        CKANMOCK,
+        interface,
+        organization_data,
+        source_data_dcatus_single_record,
+        single_internal_record,
+    ):
+        CKANMOCK.return_value.action = "ok"
+        interface.add_organization(organization_data)
+        interface.add_harvest_source(source_data_dcatus_single_record)
+        harvest_job = interface.add_harvest_job(
+            {
+                "status": "pending",
```

Comment:
"pending" is no longer a value used in the enum for harvest job status. it should either be "new" or "manual"

Reply:
Did you change that in your PR? Currently still valid here: https://github.com/GSA/datagov-harvesting-logic/blob/main/database/models.py#L53-L54